fast/stages/02-networking-nva/nva.tf

```diff
@@ -54,6 +54,13 @@ module "nva-template-ew1" {
     size  = 10
   }
   create_template = true
+  instance_type   = "f1-micro"
+  options = {
+    allow_stopping_for_update = true
+    deletion_protection       = false
+    # Creates preemptible instances, cheaper than regular one. Only suitable for testing.
+    preemptible = true
```

Comment:
I would not use preemptible, f1-micros are already super cheap and the time spent troubleshooting when they go down will probably cost more (and cause a lot of frustration). :)

Reply:
You're right - I had this intrusive thought of doing it ESPECIALLY because we're using MIGs, so unless there's a stockout everything should generally work. 
I ofc don't expect this to happen in prod.

We can probably drop the whole PR.

Reply:
Ah well spotted. ILB has its own HC but per docs our best practice is to also have autohealing on MIGs. Will add that.

Reply:
Done